leakcanary-android-core/src/main/res/layout/leak_canary_leak_row.xml

```diff
@@ -26,6 +27,9 @@
       android:id="@+id/leak_canary_row_text"
       android:layout_width="0dp"
       android:layout_weight="1"
+      android:layout_marginEnd="5dp"
+      android:layout_marginRight="5dp"
```

Comment:
Why use both right and end?

Reply:
I thought both were needed for backwards compatibility but maybe it's not now?

Reply:
You're probably right as it's API 17 and we're 16.

LeakCanary has no translation for RTL languages so there's not much value in using margingStart / End except for making stupid warnings happy

Reply:
Yeah was mostly doing it for the warning 😂